.gitignore

```diff
@@ -1 +1,2 @@
 out
+keystore
```

Comment:
I don't think this should go into the upstream .gitignore. You can modify your own global gitignore here: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c